src/components/AddNewBook.js

```diff
@@ -0,0 +1,11 @@
+export default function AddNewBook() {
+  return (
+    <section>
+      <h2>ADD NEW BOOK</h2>
+      <form>
+        <input type="text" name="book-title" placeholder="Book title" />
+        <input type="text" name="author" placeholder="Author" />
+      </form>
```

Comment:
Great work adding a form. 👏👏👏

However, the form isn't controlled. There is no action added to handle the interaction from users.
To do this, you are required to add a submit button which at the moment might not function.

*Kindly add a button to this form.*